core/src/i18n/en.yaml

```
@@ -1407,7 +1407,7 @@ persistent-volume-claims:
     match-expressions: Match expressions
     match-labels: Match labels
     operator: Operator
-    selector: Selector
+    selector: Selectors
```

```suggestion
    selector: Selector
```
K8s calls it "selector".